addons/web/static/src/core/utils/objects.js

```diff
@@ -10,3 +10,7 @@ export function shallowEqual(obj1, obj2) {
         obj1Keys.every((key) => obj1[key] === obj2[key])
     );
 }
+
+export function deepCopy(obj) {
```

Comment:
This must be documented, as the function isn't complete. It is fast, which is good, but it doesn't work on everything, so it must be explicitly written in the docstring